js/theme.js

```diff
@@ -9,6 +9,8 @@ jQuery(function ($) {
   });
 
   // Search collapse button hide if empty
+  // Deprecated v5.3, done by php if (is_active_sidebar('top-nav-search')) in header.php
+  // Remove in v6
```

Comment:
We going to do v6, or also remove it if they would release v5.4?

Reply:
Think we should ship a quick patch 5.2.3.4 soon, because https://github.com/bootscore/bootscore/pull/450 is urgent ;-)

I'm a bit worry to remove fallbacks in v5. Mostly we had to readd them in the past.

Reply:
Ah, alright. Just thinking as v6 could be maybe over a year before any release 😉